datasets/wiki-text/metadata.json

```diff
@@ -0,0 +1,121 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
```

Comment:
You'll need the following too: 

```json
"wd": "https://www.wikidata.org/wiki/",
```

I know I forgot it in COCO dataset, but it should be back in PR #66 :)

Reply:
Good one, thanks! https://github.com/mlcommons/datasets_format/pull/67/commits/3b31ad472f913f1b987e2c73fd25b4c091172759